contracts/Bond.sol

```diff
@@ -27,6 +28,8 @@ contract Bond is
 
     using FixedPointMathLib for uint256;
 
+    uint8 private _decimals;
```

Comment:
Is there a reason not to be `internal`?

Reply:
fixed

Reply:
Should I even be storing this? Or in decimals should I just return 

```solidity
    function decimals() public view override returns (uint8) {
         return paymentToken.decimals()
}
    ```